blog-server-api/src/extensions.rs

```diff
@@ -63,13 +66,23 @@ impl Resolve<Arc<Box<dyn EntityPostService>>> for ExtensionsProvider {
     }
 }
 
-pub fn make_extensions(rbatis: RBatis) -> impl ExtensionsProviderType {
+impl Resolve<Arc<Box<dyn EventBusService>>> for ExtensionsProvider {
+    fn resolve(&self) -> Arc<Box<dyn EventBusService>> {
+        self.event_bus_service.clone()
+    }
+}
+
+pub fn make_extensions(
+    rbatis: RBatis,
+    rabbit: Box<dyn EventBusService>,
```

Comment:
`Box<dyn EventBusService>` dynamic component -> different lib under the hood -> generic name for param

Reply:
Name changed